static/app/stores/tagStore.tsx

```diff
@@ -90,6 +90,12 @@ const storeConfig: TagStoreDefinition = {
         ],
         predefined: true,
       },
+      [FieldKey.ISSUE_PRIORITY]: {
```

Comment:
We should only add this when the `issue-priority-ui` flag is enabled. Looks like we have the org here so should be a simple fix